<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonScraper</class>
 <widget class="QDialog" name="PythonScraper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python Scraper</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>851</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>URL to Scrap</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>http://www.gsmarena.com/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScrap">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Scarp</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPhones">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>IPhones</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sony Mobiles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Samsung Mobiles</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>100</y>
     <width>861</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Result</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>16</y>
      <width>841</width>
      <height>501</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>839</width>
       <height>499</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="txtResult">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>6</y>
        <width>831</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
